src/Database/ClientOutboxData.sql

```diff
@@ -0,0 +1,17 @@
+﻿CREATE TABLE [dbo].[ClientOutboxData]
+(
+    [MessageId] UNIQUEIDENTIFIER NOT NULL PRIMARY KEY NONCLUSTERED,
+    [EndpointName] NVARCHAR(150) NOT NULL,
+    [CreatedAt] DATETIME NOT NULL,
+    [Dispatched] BIT NOT NULL DEFAULT(0),
+    [DispatchedAt] DATETIME NULL,
+    [Operations] NVARCHAR(MAX) NOT NULL,
+	[PersistenceVersion] VARCHAR(23) NOT NULL DEFAULT '1.0.0'
+)
+GO
+
+CREATE INDEX [IX_CreatedAt_PersistenceVersion] ON [dbo].[ClientOutboxData] ([CreatedAt] ASC, [PersistenceVersion] ASC) WHERE [Dispatched] = 0
+GO
+
+CREATE INDEX [IX_DispatchedAt_PersistenceVersion] ON [dbo].[ClientOutboxData] ([DispatchedAt] ASC, [PersistenceVersion] ASC) WHERE [Dispatched] = 1
+GO
```

Comment:
we're not using Outbox feature hence this isn't needed

Reply:
ok will remove